source/ranges.tex

```diff
@@ -2307,7 +2307,7 @@
     iterator() = default;
     constexpr iterator(Parent& parent, iterator_t<Base> current);
     constexpr iterator(iterator<!Const> i)
```

Comment:
It'd be clearer to use `iterator<false>` here. Oh, we can't do that because that'd be ill-formed when instantiating with `Const = false`. Yuck, what a language!